modules/workspace.nix

```diff
@@ -4,6 +4,7 @@
 let
   cfg = config.programs.plasma;
   inherit (import ../lib/wallpapers.nix { inherit lib; }) wallpaperPictureOfTheDayType wallpaperSlideShowType;
+  inherit (widgets.lib) stringIfNotNull;
```

Comment:
I believe you need `inherit (import ./widgets/lib.nix {inherit lib; }) stringIfNotNull` here.

Reply:
Ah, because `panels.nix` has the following import in contrast to `workspace.nix`?
https://github.com/nix-community/plasma-manager/blob/8def0842f289af5d323d14219f1edccd388b195a/modules/panels.nix#L37